package.json

```diff
@@ -9,7 +9,23 @@
     "type": "git",
     "url": "https://github.com/player-ui/rules_player.git"
   },
+  "scripts": {
+    "build:ts": "yarn esbuild --bundle --platform=node gh-pages/gh-pages.ts --outfile=gh-pages/gh-pages.js"
```

Comment:
Is there a reason this isn't included as part of a bazel build rule?

Reply:
Does this path map to anything? It looks like the script is under `gh-pages/private/gh-pages.ts`